workshops/templates/workshops/all_persons.html

```diff
@@ -17,14 +18,16 @@
 {% if all_persons %}
     <table class="table table-striped">
       <tr>
-        <th width="50">Instructor?</th>
+        <th width="25">DC?</th>
+        <th width="25">SWC?</th>
         <th>Name</th>
         <th>Email</th>
         <th class="additional-links"></th>
       </tr>
     {% for person in all_persons %}
       <tr>
-        <td class="text-center">{% if person.pk in instructors %}✓{% endif %}</td>
+        <td class="text-center">{% if person.pk in dc_instructors %}{% bootstrap_tag "DC" %}{% endif %}</td>
+        <td class="text-center">{% if person.pk in swc_instructors %}{% bootstrap_tag "SWC" %}{% endif %}</td>
```

Comment:
I thought about using `person.badges.instructor_badges()` but this yields additional query for every person in the list…